en/appendices/4-4-migration-guide.rst

```diff
@@ -112,10 +112,11 @@ ORM
 Plugins
 -------
 
-- Plugin class names now match the plugin name with a "Plugin" suffix. For
-  example, the plugin class for "ADmad/I18n" would be ``ADmad\I18n\I18nPlugin``
+- Plugin class names should now match the plugin name with a "Plugin" suffix. For
+  example, the plugin class for ``ADmad/I18n`` plugin would be ``ADmad\I18n\I18nPlugin``
   instead of ``ADmad\I18n\Plugin``, as was the case for CakePHP 4.3 and below.
-  The old style name is still supported for backwards compatibility.
+  The old style name for existing majors should be kept to avoid BC breaks. 
+  So the new convention is for new plugins or when doing a new major release.
```

Comment:
Style-wise, I think, a sentense shouldn't start with "So". Especially not in written English. But I'm not a native speaker.

Reply:
```suggestion
  The new plugin classs naming convention should be folowed when developing a new plugin or
  when doing a new major release for an existing plugin.
```

Reply:
typo classs